src/Microsoft.DotNet.Build.Tasks/PackageFiles/toolruntime.targets

```diff
@@ -13,7 +13,6 @@
 
     <ToolTargetFramework Condition="'$(ToolTargetFramework)' == ''">DNXCore,Version=v5.0</ToolTargetFramework>
     <ToolArchitecture Condition="'$(ToolArchitecture)' == ''">x64</ToolArchitecture>
```

Comment:
I searched the code bases in BuildTools, CoreFx, CoreClr, WCF, and CLI, and you're right that they aren't used in any of them. I've removed them from toolruntime.targets locally.